### PR TITLE
fix: #1190 Avatar blends into the background

### PR DESCRIPTION
**Pull request by timoxa20**
Fix Changed the background color of the Avatar component, now it does not merge with the background

### CODE REVIEW

File: mw-webapp/src/component/avatar/Avatar.tsx [hunk 1]
```diff
@@ -69,26 +70,51 @@ const getInitials = (name: string): string => {
...
-    >
-      <AvatarRadix.Image
-        className={styles.AvatarImage}
-        src={props.src ?? ""}
-        alt={initials}
-      />
-      <AvatarRadix.Fallback
-        className={styles.AvatarFallback}
-        delayMs={600}
+  if (location.pathname === "/") {
```
Comment (tritonJS826) [inline mw-webapp/src/component/avatar/Avatar.tsx:75]: super dangerous, avatar as component should not change it behavior in depends on the location

**Review by tritonJS826 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/avatar/Avatar.tsx (hunk 1 above)
Comment (Ekaterina1994) [reply mw-webapp/src/component/avatar/Avatar.tsx:75]: Agree, discused about it in telegram

File: mw-webapp/src/component/avatar/Avatar.module.scss [hunk 2]
```diff
@@ -13,6 +13,7 @@ $avatarLargeSize: 120px;
   border-radius: $tertiaryBorderRadius;
   user-select: none;
   vertical-align: middle;
+  border: 1px solid var(--primaryStrokeColor)
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/avatar/Avatar.module.scss:16]: global var instead 1px

Comment (timoxa20) [reply mw-webapp/src/component/avatar/Avatar.module.scss]: Fix border number 1px на global varieibls

**Review by Ekaterina1994 — APPROVED** [review]

**Pull request merged**